src/Form/OverrideType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @copyright Zicht Online <https://www.zicht.nl>
+ */
+namespace Zicht\Bundle\AdminBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A type utilizing the override functionality.
+ */
+class OverrideType extends AbstractType
```

Comment:
Misschien is een naam als `OverrideObject` duidelijker. Override kan van alles betekenen